src/ai/backend/manager/api/stream.py

```diff
@@ -463,11 +464,12 @@ async def stream_proxy(
     except (SessionNotFound, TooManySessionsMatched):
         raise
     kernel: KernelRow = session.main_kernel
-    kernel_id = kernel.id
-    stream_key = kernel_id
+    kernel_id = KernelId(kernel.id)
+    session_id = SessionId(session.id)
+    stream_key = session_id
     stream_id = uuid.uuid4().hex
-    app_ctx.stream_proxy_handlers[stream_key].add(myself)
-    defer(lambda: app_ctx.stream_proxy_handlers[stream_key].discard(myself))
+    app_ctx.stream_proxy_handlers[kernel_id].add(myself)
+    defer(lambda: app_ctx.stream_proxy_handlers[kernel_id].discard(myself))
```

Comment:
`stream_proxy_handlers` in other places (e.g., `handle_kernel_terminating()`) still uses `kernel.id` as the stream key. We should make the naming of `stream_key` consistent in two different usage scenarios in this module.